MeNext/net/API.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using Xamarin.Forms;
+using System.Diagnostics;
+using System.Net.Http;
+using System.Runtime.Serialization.Json;
+
+namespace MeNext
+{
+
+	public class API
+	{
+		public API(string ip)
+		{
+			this.ServerIP = ip;
+			this.Client = new HttpClient();
+
+		}
+
+		/// <summary>
+		///  SayHello is a testing method that just servers to ping the server. 
+		/// </summary>
+		/// <returns>The server response string.</returns>
+		public async System.Threading.Tasks.Task<string> SayHello()
```

Comment:
`using System.Threading.Tasks;`
...
`public async Task<string> SayHello()`

(And elsewhere)